code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/AepUi.kt

```diff
@@ -16,14 +16,14 @@ import com.adobe.marketing.mobile.aepuitemplates.AepUITemplate
 
 /**
  * Represents a UI component that can be rendered using AEP UI templates.
- * This interface can be implemented by various UI components like [SmallImageCardAepUi], [LargeImageCardAepUi], etc.
+ * This interface can be implemented by various UI components like [SmallImageUi], LargeImageUi, etc.
  * It associates a specific template and state to ensure consistency when rendering and managing UI components.
  *
  * @param T The type of the template associated with the UI component.
  * @param S The type of the state associated with the UI component.
  */
 
-sealed interface AepUI<T : AepUITemplate, S : AepCardUIState> {
+sealed interface AepUi<T : AepUITemplate, S : AepCardUIState> {
```

Comment:
This should be left as it was based on [Kotlin naming conventions for acronyms](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names)

_When using an acronym as part of a declaration name, capitalize it if it consists of two letters (IOStream); capitalize only the first letter if it is longer (XmlFormatter, HttpInputStream)._

Reply:
ah my bad, will revert the name change